CRM/Event/Page/ManageEvent.php

```diff
@@ -314,7 +314,10 @@ public function browse() {
     $this->_searchResult = CRM_Utils_Request::retrieve('searchResult', 'Boolean', $this);
 
     $whereClause = $this->whereClause($params, FALSE, $this->_force);
-    $this->pagerAToZ($whereClause, $params);
+
+    if (CRM_Core_Config::singleton()->includeAlphabeticalPager) {
```

Comment:
I wonder if we could do this check in the pagerAToZ function & not everywhere else - seems like you have found a few that got missed & hard to imagine there aren't more. Plus it would be less code duplication